package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hyperplay",
-  "version": "0.22.1",
+  "version": "0.22.2-analytics",
```

Comment:
There is a filter in the dev mixpanel for this app version so that we can see if the data only from this branch fixes the issue. I will revert to 0.22.2 before merging